services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -370,14 +370,17 @@ public static Builder builder() {
    * <p>
    * {@link DirectionsCriteria#PROFILE_CYCLING}: {@link DirectionsCriteria#EXCLUDE_FERRY}
    *
-   * @return a string matching one of the {@link DirectionsCriteria.ExcludeCriteria} exclusions
+   * Excluded points are formatted like: point(longitude latitude)
+   *
+   * @return a comma separated string where each element matches one of
+   *   the {@link DirectionsCriteria.ExcludeCriteria} exclusion or a point
    */
   @Nullable
   public abstract String exclude();
 
   /**
-   * Exclude certain road types from routing. The default is to not exclude anything from the
-   * profile selected. The following exclude flags are available for each profile:
+   * Exclude certain road types and points from routing. The default is to not exclude anything
```

Comment:
Attention ⚠️ 
How do you think, is that a logical breaking change? I have doubts about whether I need to filter out points from that filed. I think no. What do you think?

Reply:
I wouldn't consider this a breaking change. It just reflects the state of the server-side API itself which wasn't considered breaking since the format is a generic string.